code/core/src/builder-manager/utils/template.ts

```diff
@@ -46,6 +47,7 @@ export const renderHTML = async (
     files: { js: jsFiles, css: cssFiles },
     favicon: await favicon,
     globals: {
+      FLAGS: JSON.stringify(await flags, null, 2),
       FEATURES: JSON.stringify(await features, null, 2),
```

Comment:
Can you reach out to the Chromatic folks to figure out, whether they use `FEATURES` and whether this would cause a breaking change on their side?